website/management/commands/create_broken_auth_tasks.py

```
@@ -0,0 +1,277 @@
+                </ul>
+            """,
+            "mcq_question": "Which improves authentication security?",
+            "mcq_options": [
+                "A) Weak passwords + long-lived tokens",
+                "B) MFA + rate-limiting + session rotation",
+                "C) No logging",
+                "D) Reuse tokens everywhere",
+            ],
+            "correct_answer": "B",
+        },
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)